server/src/views/frame.js

```diff
@@ -222,6 +222,7 @@ class Frame extends React.Component {
 
   clickFullPageButton(e) {
     e.preventDefault();
+    history.pushState({state: "frame"}, "", "#frame");
```

Comment:
there are other parts of code that rely on the frame having an id. also, i think the inconsistent experience is better -- when the user clicks on the full page button, we want to show it to them, because they just clicked it. but, when the user visits the page with the hash, they should just see the full page